webui/src/pages/auth/groups/group/members.jsx

```diff
@@ -86,7 +74,7 @@ const GroupMemberList = ({ groupId, after, onPaginate }) => {
                     modalTitle={'Add to Group'}
                     addText={'Add to Group'}
                     resolveEntityFn={resolveUserDisplayName}
-                    searchFn={prefix => searchUsers(prefix, 5, resolveUserDisplayName).then(res => res)}
+                    searchFn={(prefix, after) => auth.listUsers(prefix, after, 5)}
```

Comment:
Searching members is different than the others - note it uses `resolveUserDisplayName`. Please revert this change, since it requires different handling.

AFAICT it should work fine since it'll just assume `after` is empty and ignore it, but please validate it still works.

Reply:
Discussed on Slack, but the short version is that `resolveUserDisplayName` helps searching also using the user's email and name, hence if removed, this functionality will be removed. So you need to find a solution for this.